styles.css

```
@@ -0,0 +1,90 @@
+* {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0;
+}
+
+body {
+  background-color: rgb(192, 244, 226);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+h1 {
+  font-size: 4rem;
+  color: rgb(4, 95, 58);
+}
+
+h2 {
+  font-size: 2rem;
+  margin-left: 25px;
+  color: rgb(20, 96, 66);
+}
+
+div {
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: 10px;
+  border-radius: 40px;
+}
+
+p {
+  font-size: 1.25rem;
+  padding: 10px;
+}
+
+ol,
+ul {
+  margin: 20px auto;
+}
+
+ol {
+  list-style-type: decimal;
+}
+
+li {
+  color: rgb(33, 31, 31);
+  font: optional;
+}
+
+img {
+  margin-top: 50px;
+  width: 200px;
+  border-radius: 50px;
+}
+
+.contenedor {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 auto;
+  margin: 0 auto;
+  background-color: rgb(178, 244, 222);
+}
+
+.titulo {
+  margin: 20px;
+  justify-content: center;
+  background-color: aquamarine;
+  text-align: right;
+  padding-right: 50px;
+  padding-left: 70px;
+  padding-bottom: 40px;
+  padding-top: 0;
+  display: flex;
+}
+
+.titulo-text {
+  padding-right: 40px;
+  padding-left: 40px;
+  padding-top: 40px;
+  border: 0;
+}
+
+.titulo-text h4 {
+  padding-top: 30px;
+}
+
+h4 spam {
+  color: brown;
+}

```

Cuidado con esto, puede que funcione pero es un abuso del lenguaje. La etiqueta correspondiente es span

```suggestion
h4 span {
  color: brown;
}

```